src/Rover.cpp

```diff
@@ -35,6 +37,7 @@ int main(int argc, char **argv)
             ParseBaseStationPacket(buffer);
         }
         autonomous.autonomyIter();
+        usleep(1000 * 1000 / CONTROL_HZ);
```

Comment:
Instead of assuming that the code executes in 0 time, do you think we can measure elapsed time and subtract that from the time to sleep? The controllers and filters would probably perform better with a more accurate dt.

Reply:
Yeah, that's a good idea. I always write it this way first because I'm lazy, but then I always end up having to change it later